lib/liquid/tags/case.rb

```diff
@@ -7,18 +7,15 @@ class Case < Block
 
     attr_reader :blocks, :left
 
-    def initialize(tag_name, markup, options)
-      super
+    def parse(tokens)
```

Comment:
Why not call `super` in here as in `Capture`? (same below)

I'm also not a fan of `super`. It couples base class and subclasses very tightly. We could address this by splitting the method in two (one for the base class, which calls another method implemented by the subclasses that doesn't need to call `super`).

Reply:
I'm not either, I've been playing a bit on how to clean this up to not need it, really these super calls are just telling it to handle the block content which should just be done anyway.